panc/src/main/scripts/panlint/tests.py

```diff
@@ -29,6 +29,32 @@ def setUp(self):
         self.maxDiff = None
         self.longMessage = True
 
+    def _assert_lint_line(self, line, diagnoses, messages, problems, first_line=False):
```

Comment:
Could you please add a docstring explaining what each of these arguments represents? e.g. looks like from usage that line is Line object, diagnoses is a list, etc.